docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,206 @@
+# Styling components
+Tizen 6.5 introduces two new methods for styling components derived from the `View` object:
+  - `ViewStyle`: designed for simple themes
```

Comment:
Can we have periods at the end of the sentences? 

Line 3-4

- `ViewStyle`: designed for simple themes.
-  XAML file: designed for advanced themes. The styles change in runtime and can be shared between the applications.

Reference: 
https://developers.google.com/style/lists?hl=en

Reply:
Done